profiles/latest/cognitiveservices/face/models.go

```diff
@@ -177,13 +177,13 @@ type IdentifyRequest = original.IdentifyRequest
 type IdentifyResult = original.IdentifyResult
 type ImageURL = original.ImageURL
 type Landmarks = original.Landmarks
+type List = original.List
 type ListDetectedFace = original.ListDetectedFace
 type ListIdentifyResult = original.ListIdentifyResult
-type ListListType = original.ListListType
+type ListList = original.ListList
```

Comment:
It is more a question on the code generation, why do we get a ListList model? Is it like that in the swagger?

Reply:
pretty much. This is what is in swagger: https://github.com/Azure/azure-rest-api-specs/blob/0bef2b99ad6d52781e3c308bc6b420f6eef59ef7/specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json#L2378